pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -283,6 +283,88 @@ def itemClass(self):
             'str': StrParameterItem,
         }[self.opts['type']]
 
+    def specific_options_from_xml(el):
+        """
+        Extract and convert a typed value from an XML element.
+
+        This function retrieves the `value` attribute from the given XML element
+        and converts it into a Python value based on the type specified in the `type` attribute.
+
+        Args:
+            el (xml.etree.ElementTree.Element): The XML element containing `type` and `value` attributes.
+
+        Returns:
+            dict: A dictionary containing the key `'value'` with the corresponding typed value.
+
+        Raises:
+            TypeError: If the type specified in the `type` attribute is unsupported.
+        """
+        param_dict = {}
+        value = el.get('value',None)
+        if el.get('type') == 'bool':
+            param_dict['value'] = True if value == '1' else False
+        elif el.get('type') == 'int':
+            param_dict['value'] = int(value)
+        elif el.get('type') == 'float':
+            param_dict['value'] = float(value)
+        elif el.get('type') == 'str':
+            param_dict['value'] = value
+        else:
+            raise TypeError(f'No interpreter found for type {el.get("type")}')
+        
+        for key in ["filetype", "detlist", "movelist", "addList", "addText", "label", "limits"]:
```

Comment:
are all these attributes present for a SimpleParameter?

Reply:
no you are right, only limits is useful!

Reply:
the other ones are related to other types